template.yaml

```diff
@@ -94,6 +101,13 @@ Parameters:
 Conditions:
   CreateUser: !Not [!Equals [!Ref EmailAddress, ""]]
   CreateCloudFrontDistribution: !Not [!Equals [!Ref CreateCloudFrontDistribution, "false"]]
+  NonSPAMode: !Equals [ !Ref EnableSPAMode, 'false']
+  CreateCloudFrontDistributionNonSPAMode: !And
```

Comment:
Why three condition actually? Think we would need just one "SpaModeEnabled"? (prefer to avoid the word Non-Spa if we can)

Reply:
I usually prefer to just put them as their own Conditions rather than do the `And` statements in the body. Happy to change these though to a single one.

Reply:
I'm attempting to do this with poor results. Full disclosure, I haven't had a lot of pure CFN exposure. Every article I find is recommending to create a separate condition that incorporates both (as I'm doing above):

https://stackoverflow.com/questions/55483955/how-to-specify-multiple-resource-conditions-in-cloudformation-template
https://stackoverflow.com/questions/50553400/multiple-conditions-on-a-single-cloudformation-resource

@ottokruse do you have any suggestions on how to accomplish this? At the moment all three conditions are needed throughout the template.

Reply:
After talking with some colleagues I think I may have just misunderstood what you were infering. I modeled the Conditionals based on the standard you have with the ReactApp, only launching it if we have a CloudFront Distribution. If that logic is not correct, and we'll launch it regardless, we can probably cut out one of the Conditionals.

Let me know if that makes sense, and if I'm understanding correctly. Happy to make adjustments with guidance.